cloaks/config.py

```diff
@@ -22,6 +24,7 @@ def load(filepath: str):
         config_yaml = yaml.safe_load(file.read())
 
     nickname = config_yaml["nickname"]
+    message  = Template(config_yaml["message"] or "")
```

Comment:
```suggestion
    message  = Template(config_yaml.get("message", ""))
```

you need to use `.get("key", "default value")` here otherwise the bot will just crash with a `KeyError` if they key doesn't exist (although that's what we do with every other mandatory config value so maybe we should just not use a default empty string value at all)